File: Simple Interest/SI.c
#include<stdio.h>
void main(){
    int prin,time,rate;
    double in=0.0,amt=0.0;
    printf("Enter the Principal : ");
    scanf("%d",&prin);
    printf("Enter the Rate of Interest : ");
    scanf("%d",&rate);
    printf("Enter the Time : ");
    scanf("%d",&time);
    in=(prin*time*rate)/100;
    amt=prin+in;
    printf("Interest : %0.2f\n",in);
    printf("Amount : %0.2f",amt);
}

File: Simple Interest/SI.java

//Calculate Simple Interest 
import java.util.Scanner;
public class SI {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Principal : ");
        int prin=sc.nextInt();
        System.out.print("Enter Rate : ");
        int rate=sc.nextInt();
        System.out.print("Enter Time : ");
        int time=sc.nextInt();
        sc.close();
        double in=(prin*rate*time)/100;
        double amt=prin+in;
        System.out.println("Interest : "+in);
        System.out.print("Amount : "+amt);
    }
}
